src/NemoStuff.jl

```diff
@@ -671,19 +671,19 @@ function gens(R::Union{Generic.ResidueRing{T},Generic.ResidueField{T}}) where {T
     return r
 end
 
-AbstractAlgebra.promote_rule(::Type{LocElem{T}}, ::Type{T}) where {T} = LocElem{T}
+promote_rule(::Type{LocElem{T}}, ::Type{T}) where {T} = LocElem{T}
```

Comment:
For the promote_rules I would rather keep this to make it more clear that AA.promote_rule and Base.promote_rule are different

Reply:
Fine by me in principle, but shouldn't it then be done in the >50 other places defining such methods? It seems a bit arbitrary to just single out this one?

Reply:
Yeah, true. Just ignore my comment then.